### ISSUE TITLE
Set-Location or any of it allies (cd) without any parameters takes you back to user directory.

### ISSUE THREAD

**Issue by adurmus** [PowerShell/PowerShell#15499]
In Windows Powershell:
![image](https://user-images.githubusercontent.com/47557218/120308013-89bdd080-c2cb-11eb-988a-ff4bf088b4f2.png)

In Powershell:

![Untitled](https://user-images.githubusercontent.com/47557218/120309753-84618580-c2cd-11eb-99d5-e732bf92ebec.jpg)

**Comment by 237dmitry** [comment]
This is normal behavior to set home location, just like in all other command shells. Except for cmd.exe, where cd outputs the path to the current directory.

**Comment by iSazonov** [comment]
Yes, this was implemented like in bash.

**Comment by jborean93** [comment]
Just as an FYI this is a change in behaviour from WinPS. Doing `Set-Location` or `cd` doesn't change any location.

**Comment by adurmus** [comment]
> This is normal behavior to set home location, just like in all other command shells. Except for cmd.exe, where `cd` outputs the path to the current directory.

This is not the case in WinPS. And I wish this change was documented somewhere from WinPS to Powershell changes etc.

**Comment by vexx32** [comment]
We have these two docs:

-  https://docs.microsoft.com/en-us/powershell/scripting/whats-new/what-s-new-in-powershell-core-60?view=powershell-7.1
- https://docs.microsoft.com/en-us/powershell/scripting/whats-new/breaking-changes-ps6?view=powershell-7.1

I'm not sure if this was done in 6.0 or later on, though, because I don't see it on this list. Do we have all the release notes in the docs site somewhere? 🤔

**Comment by 237dmitry** [comment]
> And I wish this change was documented somewhere

```
 PS > Get-Help Set-Location -parameter Path

-Path <System.String>
    Specify the path of a new working location. If no path is provided, `Set-Location` defaults to the current user's home directory.
...
```

**Comment by adurmus** [comment]
> > And I wish this change was documented somewhere
> 
> ```
>  PS > Get-Help Set-Location -parameter Path
> 
> -Path <System.String>
>     Specify the path of a new working location. If no path is provided, `Set-Location` defaults to the current user's home directory.
> ...
> ```

Is this already in place? I am confused...
```
>Get-Help Set-Location -parameter Path                                                                       ❮  

-Path <string>

    Required?                    false
    Position?                    0
    Accept pipeline input?       true (ByValue, ByPropertyName)
    Parameter set name           Path
    Aliases                      None
    Dynamic?                     false
    Accept wildcard characters?  false`
```

**Comment by vexx32** [comment]
You may need to run `Update-Help` by the looks of it.

**Comment by adurmus** [comment]
` Update-Help -UICulture en-US                                                                                ❮  
Update-Help: Failed to update Help for the module(s) 'PSReadline' with UI culture(s) {en-US} : One or more errors occurred. (Response status code does not indicate success: 404 (The specified blob does not exist.).).
English-US help content is available and can be installed using: Update-Help -UICulture en-US.
Update-Help: Failed to update Help for the module(s) 'HgsClient, HgsDiagnostics' with UI culture(s) {en-US} : A Help URI cannot contain more than 10 redirections. Specify a valid Help URI..
English-US help content is available and can be installed using: Update-Help -UICulture en-US.`

And this is known [issue](https://github.com/PowerShell/PowerShell/issues/13591).

**Comment by iSazonov** [comment]
This was implemented in #1534

**Comment by jhoneill** [comment]
This was new to me ... then I realised it's because I have re-aliased CD to PUSH-Location instead of SET-Location (I have CD- for POP-Location.)
If you prefer cd to behave in the cmd / win-PS method this work round has the benefit that your directories are stacked.
